WcaOnRails/app/models/registration.rb

```diff
@@ -105,12 +105,15 @@ def best_solve(event, type)
   end
 
   def entry_fee
-    competition.base_entry_fee
+    Money.new(
+      competition.base_entry_fee_lowest_denomination + competition_events.sum(:event_entry_fee_lowest_denomination),
```

Comment:
The competition_events.sum(:fee) doesn't work. I think the as the "fee" is created by using `monetize` it's seems rails can't sum this as it's not a database column.